blablas/ep46/index.md

```diff
@@ -0,0 +1,73 @@
+---
+date: 2020-06-21
+time: 20h:00min
+duration: "02:28:04"
+title: "Mobile Development"
+tags: ["Android", "IOS", "ReactNative", "Flutter"]
+isNext: false
+video: "272185967229889"
+published: true
+featured: false
+url: ""
+podcastUrl: https://www.facebook.com/geeksblabla/videos/272185967229889/
+---
+
+In this episode of GeeksBlabla we discuss Mobile development with some Amazing Community folks, How to get started , Native, cross-platform solution, hybrid solutions , PWA and a lot of topics around it.
+
+## Guests
+
+[Mehdi Sakout](https://twitter.com/MedyO80)
+
+[Raouf Rachiche](https://twitter.com/raoufrahiche)
+
+[Youssouf EL Azizi](https://elazizi.com/)
+
+## Notes
+
+0:00 - Introduction and welcoming
+
+0:08 - Mobile Development? History ?
+
+0:14 - Mobile Development and how it’s different from Web or desktop development (challenges).
+
+0:23 - Stat of Mobile Dev in Morocco and worldwide ( job, technologies salaries )
+
+0:47 - Solution for Mobile Development( Native, Cross-platform, hybrid, PWA)
```

Comment:
remove the space between "(" and  "Native"

Reply:
0:23 - The state of Mobile Dev in Morocco and worldwide ( jobs, technologies, salaries)